data/adoptions.yaml

```diff
@@ -23,8 +21,6 @@ main:
   tag: Flynn
 - url: https://www.youtube.com/watch?v=ofXHmfOTLyA
   tag: Quay
-- url: https://github.com/digitalocean/do-agent
-  tag: DigitalOcean
 - url: https://blog.cloudflare.com/pal-a-container-identity-bootstrapping-tool
   tag: Cloudflare
 - url: https://docs.automotivelinux.org/docs/architecture/en/dev/reference/security/05-platform-security.html#agl-platform-software-update
```

Comment:
This link is broken.

Reply:
Which link is broken? DigitalOcean was removed...